index.html

```diff
@@ -59,121 +70,285 @@ <h2>I'm a Software Developer</h2>
     <section>
         <div class="work-section">
             <div class="recent-work">
-                <h3>My Recent Work</h3>
-                <hr>
+                <h3>My Recent Works</h3>
+                <hr class="recent-hr">
```

Comment:
Kindly use the same colors as the template as requested by the previous reviewer the color of this title is darker than it should be use this `color: '#091E42'`